core/src/main/scala/stryker4s/run/MutantRunner.scala

```diff
@@ -69,11 +69,38 @@ class MutantRunner(
     val targetDir = config.baseDir / "target"
     for {
       _ <- Resource.eval(Files[IO].createDirectories(targetDir))
-      tmpDir <- Files[IO].tempDirectory(Some(targetDir), "stryker4s-", None)
+      tmpDir <- prepareTmpDir(targetDir)
       _ <- Resource.eval(setupFiles(tmpDir, mutatedFiles.toSeq))
     } yield tmpDir
   }
 
+  private def prepareTmpDir(targetDir: Path): Resource[IO, Path] = {
+    if (config.staticTmpDir) {
+      val staticTmpDir = targetDir / "stryker4s-tmpDir"
+      Resource
+        .eval(Files[IO].createDirectory(staticTmpDir).map(_ => staticTmpDir))
+        .onFinalizeCase(staticTmpDirFinalizeCase(staticTmpDir))
```

Comment:
I think what you want here is `Resource.makeCase` with the first argument being creating the directory, and the second handling deleting it when the resource is done

Reply:
Done.